src/Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -269,14 +269,24 @@ private static SKEncodedOrigin GetSKEncodedOrigin(ImageOrientation? orientation)
             }
 
             // create the bitmap
-            var bitmap = new SKBitmap(codec.Info.Width, codec.Info.Height, !requiresTransparencyHack);
+            SKBitmap? bitmap = null;
+            try
+            {
+                bitmap = new SKBitmap(codec.Info.Width, codec.Info.Height, !requiresTransparencyHack);
 
-            // decode
-            _ = codec.GetPixels(bitmap.Info, bitmap.GetPixels());
+                // decode
+                _ = codec.GetPixels(bitmap.Info, bitmap.GetPixels());
 
-            origin = codec.EncodedOrigin;
+                origin = codec.EncodedOrigin;
 
-            return bitmap;
+                return bitmap!;
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "Detected intermediary error decoding image {0}", path);
```

Comment:
Won't this get logged twice? Once here and once higher up the call stack.

Reply:
this one actually not. No "Decode" call is within any other Try-Catch